deploy/charts/openshift-routes/values.yaml

```diff
@@ -0,0 +1,146 @@
+# Default values for openshift-routes.
+
+replicas: 1
+logLevel: 5
+
+# This namespace allows you to define where the services are installed into.
+# If not set then they use the namespace of the release.
+# This is helpful when installing cert manager as a chart dependency (sub chart).
+namespace: ""
+
+# Override the "cert-manager.fullname" value. This value is used as part of
+# most of the names of the resources created by this Helm chart.
+# +docs:property
+# fullnameOverride: "my-cert-manager"
+
+# Override the "cert-manager.name" value, which is used to annotate some of
+# the resources that are created by this Chart (using "app.kubernetes.io/name").
+# NOTE: There are some inconsitencies in the Helm chart when it comes to
+# these annotations (some resources use eg. "cainjector.name" which resolves
+# to the value "cainjector").
+# +docs:property
+# nameOverride: "my-cert-manager"
+
+image:
+  # Target image registry. This value is prepended to the target image repository, if set.
+  # For example:
+  #   registry: quay.io
+  #   repository: jetstack/cert-manager-openshift-routes
+  # +docs:property
+  # registry: quay.io
+
+  # Target image repository.
+  repository: quay.io/jetstack/cert-manager-openshift-routes
```

Comment:
Previously, it was:

```yaml
repository: ghcr.io/cert-manager/cert-manager-openshift-routes
```

What's the rationale for moving to `quay.io/jetstack` by default? I'd be OK if images were both pushed to quay.io and ghcr.io, but I'm not sure I understand why we are moving from GHCR to Quay.io here